ncs/sign_script.py

```diff
@@ -178,10 +185,11 @@ def sign(self, private_key_path: Path) -> None:
     parser = ArgumentParser()
     parser.add_argument("--input-file", required=True, type=Path, help="Input envelope.")
     parser.add_argument("--output-file", required=True, type=Path, help="Output envelope.")
+    parser.add_argument("--private-key", required=True, type=Path, help="Private key.")
```

Comment:
This change adds an additional required parameter.
If this script is not used by anyone, than I'm fine with it. If not, maybe adding a default value (`PRIVATE_KEY`) could be a better option (for the sake of backward compatibility) ?

Reply:
I was thinking about this, but I would like to avoid having a fallback option to prevent a situation, where something gets signed with the wrong key.
Not sure if it's really a big problem, but if you think it's better to keep backward compatibility then I can bring this back.

Reply:
Removed the change.